CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera/CameraViewController.swift

```diff
@@ -41,6 +42,7 @@ final class CameraViewController: UIViewController {
     private var resetQRCodeTask: DispatchWorkItem?
     private var hideQRCodeTask: DispatchWorkItem?
     private var validQRCodeProcessing: Bool = false
+    private var isOnScreen = false
```

Comment:
It checks if the CameraViewController is fully on screen, and isn’t in state of appearing/disappearing (before viewDidAppear and after viewWillDisappear it’s false). It’s used to prevent processing of QR code in process of help screen being presented